src/EditorFeatures/Test/Diagnostics/TestDiagnosticAnalyzerService.cs

```diff
@@ -26,8 +27,9 @@ internal TestDiagnosticAnalyzerService(string language, ImmutableArray<Diagnosti
             ImmutableDictionary<string, ImmutableArray<DiagnosticAnalyzer>> analyzersMap,
             AbstractHostDiagnosticUpdateSource hostDiagnosticUpdateSource = null,
             Action<Exception, DiagnosticAnalyzer, Diagnostic> onAnalyzerException = null,
-            IDiagnosticUpdateSourceRegistrationService registrationService = null)
-            : this(CreateHostAnalyzerManager(analyzersMap, hostDiagnosticUpdateSource), hostDiagnosticUpdateSource, onAnalyzerException, registrationService)
+            IDiagnosticUpdateSourceRegistrationService registrationService = null,
+            IAsynchronousOperationListener listener = null)
```

Comment:
usually in unit test, async listener is not used. so other user of this type, doesnt care about listener.